semconv/src/main/scala/org/typelevel/otel4s/semconv/resource/attributes/ResourceAttributes.scala

```diff
@@ -28,15 +28,17 @@ object ResourceAttributes {
 
   /** The URL of the OpenTelemetry schema for these keys and values.
     */
-  val SchemaUrl: String = "https://opentelemetry.io/schemas/1.19.0"
+  final val SchemaUrl: String = "https://opentelemetry.io/schemas/1.19.0"
 
   /** Array of brand name and version separated by a space
     *
     * <p>Notes: <ul> <li>This value is intended to be taken from the <a
     * href="https://wicg.github.io/ua-client-hints/#interface">UA client hints
     * API</a> ({@code navigator.userAgentData.brands}).</li> </ul>
     */
-  val BrowserBrands: AttributeKey[List[String]] = stringList("browser.brands")
+  final val BrowserBrands: AttributeKey[List[String]] = stringList(
+    "browser.brands"
+  )
```

Comment:
Because these are instances of objects, adding `final` actually doesn't make a difference, they cannot be inlined anyway.